lib/GithubUtils.js

```diff
@@ -0,0 +1,76 @@
+export const GithubOwner = 'Expensify';
+export const ExpensifyIssueRepo = 'Expensify';
+export const ExpensifyCashRepo = 'Expensify.cash';
+
+export default class GithubUtils {
+    /**
+     * @param {Octokit} octokit - Authenticated Octokit object https://octokit.github.io/rest.js
+     */
+    constructor(octokit) {
+        this.octokit = octokit;
+    }
+
+    /**
+     * Finds one open `StagingDeployCash` issue via GitHub octokit library
+     * @returns {Promise}
+     */
+    getStagingDeployCash() {
+        return this.octokit.issues.listForRepo({
+            owner: GithubOwner,
+            repo: ExpensifyIssueRepo,
+            labels: 'StagingDeployCash',
+            state: 'open'
+        })
+            .then(({data}) => {
+                if (!data.length) {
+                    throw new Error('Unable to find StagingDeployCash issue.');
+                }
+
+                if (data.length > 1) {
+                    throw new Error('Found more than one StagingDeployCash issue.');
+                }
+
+                return this.getStagingDeployCashData(data[0]);
+            });
+    }
+
+    /**
+     * Takes in a GitHub issue object and returns the data we want
+     *
+     * @param {Object} issue
+     * @returns {Promise}
+     */
+    getStagingDeployCashData(issue) {
+        try {
+            const versionRegex = /`([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9]*))?`/g;
+            const tag = issue.body.match(versionRegex)[0].replace(/`/g, '');
+
+            // eslint-disable-next-line max-len
+            const compareURLRegex = /https:\/\/github\.com\/Expensify\/Expensify.cash\/compare\/([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9]*))?...([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9]*))?/g;
```

Comment:
I think if you declare `versionRegex` like `new RegExp('your_regex_here', 'g')`, then you can combine regexes. Here you would use it like this:

```javascript
const compareURLRegex = new RegExp(`https:\/\/github\.com\/Expensify\/Expensify\.cash\/compare\/${versionRegex.source}\.\.\.${versionRegex.source}`, 'g');
```

Also, you've got some unescaped `.` characters in here that I don't think you want (such as in `Expensify.cash` and for the `...`)